src/main/java/com/zaxxer/nuprocess/linux/LibEpoll.java

```diff
@@ -34,21 +34,21 @@
 
    public static native int epoll_ctl(int epfd, int op, int fd, EpollEvent event);
 
-   // technically, events should be an array, but we only ever call with maxevents = 1
+   // when passing "maxevents" >1, the first element in an EpollEvent[] of matching length should be
+   // passed as "events". create the array with new EpollEvent().toArray (and cast the result)
    public static native int epoll_wait(int epfd, EpollEvent events, int maxevents, int timeout);
 
    public static final int SIGPIPE = 13;
 
    /* from /usr/include/sys/epoll.h */
-   public static final int EPOLL_CTL_ADD = 1; /* Add a file decriptor to the interface.  */
-   public static final int EPOLL_CTL_DEL = 2; /* Change file decriptor epoll_event structure.  */
-   public static final int EPOLL_CTL_MOD = 3; /* Remove a file decriptor from the interface.  */
+   public static final int EPOLL_CTL_ADD = 1; /* Add a file descriptor to the interface.  */
+   public static final int EPOLL_CTL_DEL = 2; /* Remove a file descriptor from the interface.  */
+   public static final int EPOLL_CTL_MOD = 3; /* Change file descriptor epoll_event structure.  */
```

Comment:
The comments on `EPOLL_CTL_MOD` and `EPOLL_CTL_DEL` were backwards, and all three had "descriptor" misspelled